Game/Levels/Tutorial/L02rw.lean

```diff
@@ -164,7 +164,7 @@ your list of assumptions. Lean thinks of `h` as being a secret proof of the
 assumption, rather like `x` is a secret number.
 
 Before we can use `rfl`, we have to \"substitute in for $y$\".
-We do this in Lean by *rewriting* the proof `h`,
+We do this in Lean by *rewriting* the assumption `h`,
```

Comment:
```suggestion
We do this in Lean by *rewriting* the proof with `h`,
```

would this be a better phrasing?

Reply:
Yeah, that would be much clearer in my opinion.